ges/Example_GES/src/main.cpp

```diff
@@ -1,16 +1,23 @@
 #include <mbed.h>
 #include "DBS_pindefs.h"
-#include "objectlist.h"
+// Include slave-specific utypes.h before Ethercat.h
+#include "utypes.h"
 #include "Ethercat.h"
 
 #define WAIT_TIME (2) // seconds
-#define APP_TITLE "Example GES"
-#define PC_BAUDRATE (128000)
+#define APP_TITLE "Example GES" // Application name to be printed to terminal
+#define PC_BAUDRATE (128000) // per second
+
+// Easy access to PDOs. Needs to be changed if different PDOs are used
```

Comment:
Changed to what?

Reply:
Application-specific stuff. E.g. in temperature sensor application, the temperature pdo variables (like measured temperature) you want to access in main.